docs/DeveloperGuide.adoc

```diff
@@ -782,15 +937,94 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
+Our SLAP aims to target social workers who have to visit different less privileged families and/or high-risk
+stay-at-home patients as their daily routine. Social workers regularly make visits to these beneficiaries to assist them
+ in overcoming obstacles in their lives.
+
+Our SLAP will allow them to organise their schedule for the day as they may have to visit multiple locations, take
+copious notes and help in many different ways. For example, a social worker may have to pick a victim of domestic abuse
+up in the morning to take her to the lawyer’s office to file for a divorce from her abusive husband and after taking her
+ home, the social worker might have to visit another family to check on a disabled patient.
+
+
 * has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
+* handles sensitive contacts
+* plans and manages their daily visits on desktop
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*:
+
+* manage contacts faster than a typical mouse/GUI driven app
+* easily add and manage calendar events without a full-blown application experience
+* organize key meetings and keep track of key contacts in one place
+* able to provide routes from one location to another
+* uses minimal system resources
+* more secure than a traditional address book
+
+*Feature contribution:*
+
+**Jason Manson-Hing**
+
+`Major Feature:` Calendar Integration
+* Functional calendar within the application with add/view/delete/edit functionality.
+* This is useful for social workers to keep track of all of the things that they have to do now and in the future.
+This calendar will be seperate from their personal calendar so as to keep business and personal information seperate.
+Additionally, this calendar will integrat with the daily scheduler function to help optimize their time in a day.
+
+`Minor Feature:` Daily Scheduler (Events)
+* Collates the user's list of calendar events for the day into an easily consumable view.
+* Allows the user to plan the most optimal route of travel for their beneficiary visits for the day and creates the
+optimal schedule for them.
+
+`Minor Feature:` View Error Log in App
+* Allows the user to view the application's error from within the application itself.
+* This is useful as it allows an advanced user to send the application developer a log of the events that occurred before an exception was encountered.
+* This feature is unobtrustive as the information contained within the log is essentially meaningless to everyone except for the developer. Additionally, the command is relatively hard to invoke accidentally, and thus has a low chance of being discovered. Additionally, this command only shows read-only information and thus cannot cause any harm to the system.
```

Comment:
What I mean is that this feature is hard to accidentally call, so there's a low chance of a user encountering the error-log by accident. However, this feature is useful so that if the user encounters an issue and wishes to help provide more information, they can relatively easily retrieve this information without much effort provided they are instructed on how to do so.